pandas/tests/series/indexing/test_setitem.py

```diff
@@ -126,6 +126,14 @@ def test_setitem_boolean_different_order(self, string_series):
 
         tm.assert_series_equal(copy, expected)
 
+    def test_setitem_boolean_using_python_list(self):
+        # GH#19406
+        series = Series([None, "b", None])
```

Comment:
Here in the test you have only ``None``, but there are also ``np.nan``, ``pd.NA``, ``pd.NaN``, ``pd.NaT`` (probably something else?).
I guess it should work with them too, shouldn't it?

Reply:
I was wondering that. Will do.

Reply:
Maybe once I resolve @arw2019 comment about hardcoding the mask instead of calling isnull() that would be clearer?

Reply:
Sounds good.